README.md

```diff
@@ -1 +1,35 @@
 # GCC resources
+
+Ce repository contient tous les travaux pratiques [des stages Girls Can 
```

Comment:
s/repository/dépôt ?

Reply:
This message was brought to you by [bitoduc.fr](https://bitoduc.fr/#ang-fra-R)